functions.php

```diff
@@ -485,4 +485,12 @@ function nightingale_check_author( $display_name ) {
     }
     return $display_name;
 }
-add_filter( 'the_author', 'nightingale_check_author', PHP_INT_MAX, 1 );
\ No newline at end of file
+add_filter( 'the_author', 'nightingale_check_author', PHP_INT_MAX, 1 );
```

Comment:
PHP_INT_MAX...? why not a static integer?

Reply:
It's a way to guarantee top priority, but I can chance it to something like 9999 for simplicity.

Reply:
It guarantees top priority, but I've changed it to 9999 to avoid confusion.